src/app/styles/layout/_catalog-grid.less

```diff
@@ -1,12 +1,14 @@
 .l-product-grid {
     // use flexbox for layout on small+ screens
     // add flex display to product grid wrapper
+    margin: 0 -5px;
     @media (min-width: @screen-sm-min) {
         .oc-flex-layout.horizontal;
         .oc-flex-layout.wrap;
         // flex each item within the grid wrapper
         &__item {
             .oc-flex-layout.vertical;
+            padding: 0 5px;
```

Comment:
specifically declare {padding-left:5px;padding-right:5px} since that is what we are overriding